<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangFormat::ClangFormatGlobalConfigWidget</class>
 <widget class="QWidget" name="ClangFormat::ClangFormatGlobalConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>ClangFormat global setting:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="formattingModeLayout">
        <item>
         <widget class="QLabel" name="formattingModeLabel">
          <property name="text">
           <string>Formatting mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="indentingOrFormatting"/>
        </item>
        <item>
         <spacer name="formattingModeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="formatWhileTyping">
        <property name="text">
         <string>Format while typing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="formatOnSave">
        <property name="text">
         <string>Format edited code on file save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
